src/clnt_vc.c

```diff
@@ -236,11 +236,11 @@ clnt_vc_ncreatef(const int fd,	/* open file descriptor */
 	ct->ct_closeit = false;
 	ct->ct_wait.tv_usec = 0;
 	ct->ct_waitset = false;
-	ct->ct_addr.buf = mem_alloc(raddr->maxlen);
 
-	memcpy(ct->ct_addr.buf, raddr->buf, raddr->len);
-	ct->ct_addr.len = raddr->len;
-	ct->ct_addr.maxlen = raddr->maxlen;
+	if (sizeof(struct sockaddr_storage) >= raddr->len) {
```

Comment:
What do you suggest? This field is only copied for debug messages, and getting the value via the control. It's otherwise pointless. We should assert? Return NULL?